about.html

```diff
@@ -0,0 +1,102 @@
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+           </div>        
+      </nav>
+    </header>
+  
+    <section class="introduction">
+      <div class="intro-div">
+      <P class="greeting">Hello! Its time for </P>
+      <h2 class="welcome"> Margaret's Tech World <br />Global Summit 2023!</h2>
+      <img class="underline-img" src="./images/title_underline.svg" alt="underline">
+      <div class="intro-paragraph">
+        <p>We are thrilled to present our annual Global Summit, a convergence of visionary leaders, industry experts, and passionate enthusiasts from around the world. This summit is a platform that celebrates the transformative power of technology and its potential to shape the future.</p>
+      </div>
+      <P class="address">Please contact us as per Email for further questions about Margaret's Tech World Global Summit 2023</P>
+      <p class="email">margarettechworld@gmail.com</p>
+    </div>
+    </section>
+  </div>
+  <section class="about-logo">
+    <h3 class="about-title">Margaret's Tech World Global Summit 2023 Logo</h3>
+    <img class="underline-img" src="./images/title_underline.svg" alt="underline">
+    <p class="about-p">The logo for Margaret's Tech World (MTW) is a visually striking representation of the organization's brand identity. </p>
+    <p class="desktop-about-p">The logo for Margaret's Tech World (MTW) is a visually striking representation of the organization's brand identity. The logo features the initials "MTW" creatively intertwined or stylized in a distinctive and memorable way.</p>
+    <div class="logo-display">
+    <P class="logo about-logo-name">MTW</P>
+    </div>
+  </section>
+<section class="see-past">
+  <h3 class="see-past-title">See the past MTW Summits</h3>
+  <img class="underline-img" src="./images/title_underline.svg" alt="">
+  <p class="see-about-p">Take a look at the two last summit which took place in Bintumani and Hill Valey</p>
+  <div class="see-past-img-div">
+    <img class="see-past-img" src="./images/about_2011.png" alt="2011 past event">
+    <img class="see-past-img" src="./images/about_2013.png" alt="2013 past event">
+  </div>
+</section>
+
+<section class="partners">
+  <div class="partner-title-div">
+    <h3 class="partner-title">Partner</h3>
+    <img class="underline-img" src="./images/title_underline.svg" alt="underline">
+  </div>
+<img class="partner-img" src="./images/mozilla.svg" alt="mozilla">
+<img class="partner-img" src="./images/googlekorea.svg" alt="googlekorea">
+<img class="partner-img" src="./images/naver.svg" alt="naver">
+<img class="partner-img" src="./images/daumkakaologo.svg" alt="daumkakaologo">
+<img class="partner-img" src="./images/airbnb.svg" alt="airbnb">
+</section>
+<footer >
+  <div class="footer-logo">
+    <p class="ft-logo">MTW</p>
+  </div>
+  <div class="reserved">
+    <p>2023 Margaret Tech World. Some Rights Reserved</p>
+  </div>
+</footer>
```

Comment:
- [ ] It is required to stick to the guiding template as much as possible. In the about page. There's only dark background, but your design includes dark and white backbground on the desktop version. Kindly remove the white part of it.

### Your Design
![your-page](https://github.com/MargaretTechworld/Capstone-Project-Module1/assets/26597432/e6b59de8-92a1-4d12-8703-c5699bc10944)

### Guiding Template
![temp](https://github.com/MargaretTechworld/Capstone-Project-Module1/assets/26597432/4ac95ca0-5463-4e75-93fc-76e5a139e232)